terraform/shared/scripts/setup.sh

```diff
@@ -110,8 +109,24 @@ sudo apt-get install -y docker-ce
 
 # Needs testing, updating and fixing
 if [[ ! -z ${INSTALL_NVIDIA_DOCKER+x} ]]; then 
+
+  # key was rotated, see https://forums.developer.nvidia.com/t/notice-cuda-linux-repository-key-rotation/212771
+  sudo apt-key del 7fa2af80
+  # auto install - FAILS
+  # wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-keyring_1.0-1_all.deb
+  # sudo dpkg -i cuda-keyring_1.0-1_all.deb
+
+  # also add gnupg-curl, otherwise "gpgkeys: protocol `https' not supported"
+  sudo apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/3bf863cc.pub
+  
+  # distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
```

Comment:
Is this leftover debug code?  If so, can we delete?

Reply:
Yup, the comments can be deleted. Just some leftovers from multiple iterations before finding a working solution.